### PR TITLE
Week 3 walkthrough 4+5 merge with master (All has been approved already, just needs a sign off here)

### PR DESCRIPTION
**Pull request by mmurray22**
Updates in response to Week 3 Walkthrough 3 (incorporates Week 3 Step 4 changes)

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -63,11 +63,11 @@ function createMap() {
...
-        for (comment of comments) {
-            bodyElement.innerHTML += comment + '\n';
-        }
+	  const bodyElement = document.body;
+          console.log('Comments: ', comments);
+          bodyElement.innerHTML += 'Here are all the comments: <br>';
+          for (comment of comments) {
+    		bodyElement.innerHTML += comment + '<br>';
+          }
 	});
```
Comment (ihsan314) [inline portfolio/src/main/webapp/script.js]: Mixing tabs and spaces is messy, you should either stick with the two-space indents or stick with tabs. Also in the other functions you had three-space indents.

**Review by ihsan314 — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/script.js]: yup the tabs and spaces have been messed up from the beginning unfortunately. I use a mix of vim and editor so sometimes stuff gets wonky. I believe Chris is looking for a resource to assist with the spacing stuff :( Sorry about the inconvenience! Do you think we should block on this?

**Review by ihsan314 — APPROVED** [review]
Oh I see, yea that makes sense. What you can do in the meantime is either add this to your vimrc: `set cindent shiftwidth=2` where 2 can be replaced by the default indent of the editor, or run that command as you are editing. Then run these commands: `gg` and `=G`, which should reindent the file.

For now, I'll approve so that you can move on, the spacing can be fixed later.

**Review by ccondit — CHANGES_REQUESTED** [review]
I'm slightly confused by this PR - didn't we already approve these changes in an earlier PR? Can't you just merge that back into the main branch?

**Comment by mmurray22** [comment]
yeah I was confused too...so I merged these in with the Week 3 Walkthrough 3 branch but then that branch was blocked from merging with the master branch...

**Review by ccondit — APPROVED** [review]
I see. See step 5 [here](https://docs.google.com/document/d/19pB7lV4ErEeHUKKwqq8y9ha8QPYNZhHNkNggzdZU_os/edit#). I think we want to merge back to the main branch at the highest level first, and then work down to the children

**Comment by kaveh096** [comment]
> I'm slightly confused by this PR - didn't we already approve these changes in an earlier PR? Can't you just merge that back into the main branch?

+1 . I cannot tell what's new in this PR.

**Review by kaveh096 — APPROVED** [review]

**Pull request merged**